catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/CreateOperations.java

```diff
@@ -559,4 +474,171 @@ private CreateResponse validateFixCreateResponse(CreateResponse createResponse,
         return createResponse;
     }
 
+    private CreateResponse processPostIngestPlugins(CreateResponse createResponse) {
+        for (final PostIngestPlugin plugin : frameworkProperties.getPostIngest()) {
+            try {
+                createResponse = plugin.process(createResponse);
+            } catch (PluginExecutionException e) {
+                LOGGER.info("Plugin processing failed. This is allowable. Skipping to next plugin.",
+                        e);
+            }
+        }
+        return createResponse;
+    }
+
+    private CreateResponse performRemoteCreate(CreateRequest createRequest,
+            CreateResponse createResponse) {
+        if (!opsCatStoreSupport.isCatalogStoreRequest(createRequest)) {
+            return createResponse;
+        }
+
+        CreateResponse remoteCreateResponse = doRemoteCreate(createRequest);
+        if (createResponse == null) {
+            createResponse = remoteCreateResponse;
+        } else {
+            createResponse.getProperties()
+                    .putAll(remoteCreateResponse.getProperties());
+            createResponse.getProcessingErrors()
+                    .addAll(remoteCreateResponse.getProcessingErrors());
+        }
+        return createResponse;
+    }
+
+    private CreateResponse getCreateResponse(CreateRequest createRequest) throws IngestException {
+        // Call the create on the catalog
+        LOGGER.debug("Calling catalog.create() with {} entries.",
+                createRequest.getMetacards()
+                        .size());
+        if (!Requests.isLocal(createRequest)) {
+            return null;
+        }
+
+        CreateResponse createResponse = sourceOperations.getCatalog()
+                .create(createRequest);
+        createResponse = historian.version(createResponse);
```

Comment:
Not familiar with how the `historian` works, but would it be better to have that call in the main `create()` method instead of "hidden" here?